common/djangoapps/util/tests/test_milestones_helpers.py

```diff
@@ -4,6 +4,11 @@
 
 from mock import patch
 from util import milestones_helpers
+
+from milestones.exceptions import (
```

Comment:
I see you imported these exceptions into the test, but are not using them in the changes you made -- are they necessary?